core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3616,6 +3616,11 @@ public String toString() {
           .setDescription("The port for Alluxio job worker's RPC service.")
           .setDefaultValue(30001)
           .build();
+  public static final PropertyKey JOB_WORKER_THREADPOOL_SIZE =
+          new Builder(Name.JOB_WORKER_THREADPOOL_SIZE)
+                  .setDescription("Number of threads in the thread pool for job worker.")
```

Comment:
Could you add some more info about why a user might want to configure this? For example, # threads is roughly proportional to # of cores this will use?